Orange/widgets/data/owpythonscript.py

```diff
@@ -655,8 +660,15 @@ def commit(self):
         self.console.new_prompt(sys.ps1)
         for out in self.outputs:
             signal = out.name
-            self.send(signal, self.console.locals.get(signal, None))
-
+            out_var = self.console.locals.get(signal, None)
+            if not isinstance(out_var, out.type) and out_var is not None:
+                self.Error.add_message(signal, msg=self.error_msg(signal, out.type))
+                getattr(self.Error, signal)()
+                out_var = None
+            self.send(signal, out_var)
+
+    def error_msg(self, a, b):
```

Comment:
I suggest removing this method. However, don't use `a` and `b` for names of arguments.